.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: Vertical Collection CI
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  test:
+    name: Run Tests
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node 12
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: 12.x
+
+      - name: Node Modules Cache
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ci-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Run Tests
+        run: yarn test:ember
+        env:
+          CI: true
+
+  test-ember-try:
+    name: Run Tests
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        ember-version:
+          [
+            # The test suite only support Ember 3.x
+            # ember-lts-2.8,
```

Comment:
Should we uncomment these, if we intend to support 2.8+?

Reply:
Now that `.travis.yml` is removed, I think so 🤔

Reply:
There is a comment right above them. The test suite doesn't run in Ember 2.x. The commented lines and comment above them explain that these are disabled due to that flaw.

We should get the tests to pass on 2.x, but I'm don't think we should block it since the CI suite today on master is busted generally.

Reply:
It looks like we need to set ember-compatibility-helpers to 1.2.1 https://github.com/pzuraq/ember-compatibility-helpers/issues/47

Reply:
This PR should not be blocked on fixing the 2.x test suite. I'm happy to entertain PRs against the beta or against master after this lands.

The code has copious coverage by ember-table's suite (which is running against 2.x) an against a massive codebase internally at Addepar. If is far more important that we get master into a functional state and get a release than it is to golf out the final legs of test suite repair.